Hyperrail/src/main/res/values-fr/strings.xml

```diff
@@ -1,128 +1,136 @@
+    <string name="facility_ticket_vending_machines">Distributeurs de titres de transport</string>
+    <string name="facility_bus_stop">Bus</string>
+    <string name="facility_tram_stop">Tram</string>
+    <string name="facility_subway_stop">Métro</string>
+    <string name="facility_taxi_stop">Taxi</string>
+    <string name="facility_bicycle_parking">Parking à vélos</string>
+    <string name="facility_blue_bikes">Vélo Bleu</string>
+    <string name="title_station_opening_hours">Heures d\'ouverture</string>
+    <string name="warning_offline">Vous êtes hors-ligne. Les résultats ne sont peut-être plus à jour.</string>
+    <string name="tap_to_retry">Erreur de chargement. Touchez pour réessayer.</string>
+    <string name="title_rate">Évaluez l\'application !</string>
+    <string name="title_beta_test">Tester la version bêta</string>
+    <string name="menu_group_feedback">Feedback</string>
+    <string name="previous">Précedent</string>
+    <string name="next">Suivant</string>
+    <string name="finish">Terminer</string>
+    <string name="skip">Ignorer</string>
+    <string name="station_details_not_available_description">Les informations détaillées ne sont pas disponible pour cette gare</string>
+    <string name="station_details_not_available_title">Informations de la gare indisponibles</string>
+    <string name="station_not_found">Gare non trouvée</string>
+    <string name="widget_error_cant_load">Les données ne peuvent pas être chargées.\n Supprimez ce widget et réessayez.</string>
 
-  <string name="dialog_title_shortcut_name">Nom du raccourci</string>
-  <string name="button_add">Ajouter</string>
-  <string name="button_cancel">Annuler</string>
+    <string name="dialog_title_shortcut_name">Nom du raccourci</string>
+    <string name="button_add">Ajouter</string>
+    <string name="button_cancel">Annuler</string>
+
+    <string name="placeholder_route_suggestions">Vos itinéraires récents et favoris apparaîtront ici. Choisissez deux stations et appuyez sur le miroir pour faire votre première recherche.</string>
+    <string name="placeholder_vehicle_suggestions">Vos véhicules récents et favoris apparaîtront ici. Entrez un code de véhicule et appuyez sur le miroir pour faire votre première recherche.</string>
+    <string name="placeholder_liveboard_suggestions">Vos gares récentes, favorites et à proximité apparaîtront ici. Commencez à taper et choisissez une gare pour effectuer votre première recherche.</string>
+    <string name="error_servererror_unavailable">Désolé, le serveur n\'est pas disponible pour le moment. Nous réparons cela dès que possible.</string>
+    <string name="error_network_connected_unavailable">Vous êtes connecté à un réseau, mais vous ne semblez pas avoir de connexion avec le monde extérieur.</string>
+    <string name="error_network_unsafe">Une connexion SSL sécurisée n\'a pas pu être établie. Votre connexion Wi-Fi peut être interceptée!</string>
```

Comment:
@Whaxion do you have feedback or remarks on these new error messages and placeholder messages?